mix.exs

```diff
@@ -5,8 +5,8 @@ defmodule ZenMonitor.Mixfile do
     [
       app: :zen_monitor,
       name: "ZenMonitor",
-      version: "1.1.1",
-      elixir: "~> 1.5",
+      version: "1.2.0",
```

Comment:
I think we should rev the Major here since we are dropping some elixir versions (no problem with dropping those versions given how old they are).